winui/Cartesian-Charts/Bar.md

```diff
@@ -0,0 +1,119 @@
+
+    <chart:SfCartesianChart IsTransposed="True">
+
+        <chart:SfCartesianChart.XAxes>
+            <chart:CategoryAxis />
+        </chart:SfCartesianChart.XAxes>
+
+        <chart:SfCartesianChart.YAxes>
+            <chart:NumericalAxis />
+        </chart:SfCartesianChart.YAxes>   
+
+        <chart:SfCartesianChart.Series>
+            <chart:ColumnSeries ItemsSource="{Binding Data}" 
+                                XBindingPath="XValue" 
+                                YBindingPath="YValue"/>
+        </chart:SfCartesianChart.Series>
+
+    </chart:SfCartesianChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+    SfCartesianChart chart = new SfCartesianChart();
+    chart.IsTransposed = true;
+    CategoryAxis xAxis = new CategoryAxis();
+    chart.XAxes.Add(xAxis);
+    NumericalAxis yAxis = new NumericalAxis();
+    chart.YAxes.Add(yAxis);
+
+    ColumnSeries series = new ColumnSeries()
+    {
+        ItemsSource = new ViewModel().Data,
+        XBindingPath = "XValue",
+        YBindingPath = "YValue",
+    };
+
+    chart.Series.Add(series);
+    this.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Bar chart type in WinUI Chart](Chart-Types_images/WinUI_bar_chart.png)
+
+## Segment spacing
+
+The [SegmentSpacing](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.ColumnSeries.html#Syncfusion_UI_Xaml_Charts_ColumnSeries_SegmentSpacing) property is used to change the spacing between two segments. The default value of spacing is 0, and the value ranges from 0 to 1. Here, 1 and 0 correspond to 100% and 0% of the available space, respectively.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+    <chart:SfCartesianChart IsTransposed="True">
+
+        <chart:SfCartesianChart.XAxes>
+            <chart:CategoryAxis />
+        </chart:SfCartesianChart.XAxes>
+
+        <chart:SfCartesianChart.YAxes>
+            <chart:NumericalAxis />
+        </chart:SfCartesianChart.YAxes> 
+
+        <chart:SfCartesianChart.Series>
+            <chart:ColumnSeries ItemsSource="{Binding Data}" 
+                                XBindingPath="XValue" 
+                                YBindingPath="YValue" 
+                                SegmentSpacing="0.5"/>
+        </chart:SfCartesianChart.Series>
+
+    </chart:SfCartesianChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+    SfCartesianChart chart = new SfCartesianChart();
+    chart.IsTransposed = true;
+    CategoryAxis xAxis = new CategoryAxis();
+    chart.XAxes.Add(xAxis);
+    NumericalAxis yAxis = new NumericalAxis();
+    chart.YAxes.Add(yAxis);
+
+    ColumnSeries series = new ColumnSeries()
+    {
+        ItemsSource = new ViewModel().Data,
+        XBindingPath = "XValue",
+        YBindingPath = "YValue",
+        SegmentSpacing = 0.5
+    };
+
+    chart.Series.Add(series);
+    this.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Bar segment spacing in WinUI Chart](Chart-Types_images/WinUI_bar_chart_segment_spacing.png)
+
```

Comment:
Add the note section for bar chart and ensure other chart types demo link.

Reply:
Added note for the bar chart demo link and changed the demo links for other chart types.